harvard-lts/mps-viewer
# Moving help to separate package plugin.

**Moving help to separate package plugin.**
* * *

**JIRA Ticket**: None

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

**Documentation I found helpful:**

1. https://github.com/ProjectMirador/mirador/wiki/M3---Creating-a-Mirador-plugin
2. https://levelup.gitconnected.com/publish-react-components-as-an-npm-package-7a671a2fb7f
3. https://docs.npmjs.com/creating-and-publishing-an-organization-scoped-package

**New Plugin Locations:**

1. https://github.com/harvard-lts/mirador-help-plugin
2. https://www.npmjs.com/package/@harvard-lts/mirador-help-plugin

# What does this Pull Request do?
This takes the `harvardHelp` plugin out of the mps-viewer repository and puts it in a separate repository / npm package so that other people in the Mirador community can install it separately in their own viewers. 

I created the (public) github repository noted above along with the (public) npm package also noted above. The npm package is in a new `harvard-lts` organization. I created a `developers group there, so everyone relevant should create npm accounts, then I can add them to the organization there.

Then I updated the package.json file in mps-viewer to use the npm package. The functionality should be exactly the same: The navigation / pager is not displayed on single images.

# How should this be tested?

A description of what steps someone could take to:
* Completely rebuild the mps-viewer container using the `move-help-plugin` branch
* npm install should successfully install @harvard-list/mirador-help-plugin in the node_packages folder and webpack should successfully build without errors.
* If you view any items, the help modal window should still display: https://localhost:23017/example/W401849_URN-3:HUL.ARCH:2009749
* 
# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz
